engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -120,14 +126,36 @@ object MaterializeWorkflowDescriptorActor {
         }
       }
 
+      val errorOrMaybePrefixes = workflowOptions.getVectorOfStrings("call_cache_hit_path_prefixes")
+      val errorOrInvalidateBadCacheResults = errorOrCallCachingBoolean("invalidate-bad-cache-results")
```

Comment:
Seems weird to treat a missing config value as a per-workflow validation error rather than applying a default?

Reply:
Nothing's ever missing AFAIK, and the newly added tests validate this.

The defaults are still there in the `reference.conf` (the belt). I just removed the duplicated defaults that were never used by main scala (the suspenders).